ui/00_message/common/furniture_layout_name.xml

```diff
@@ -167,45 +171,56 @@
   <message>
     <key>FURNITURE_LAYOUT_NAME_39</key>
     <original>壁紙</original>
+    <translation>Wallpaper</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_40</key>
     <original>湯口</original>
+    <translation>Spout</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_41</key>
     <original>鳩時計</original>
+    <translation>Cuckoo Clock</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_42</key>
     <original>壁面家具・左</original>
+    <translation>Left Wall Furniture</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_43</key>
     <original>壁面家具・右</original>
+    <translation>Right Wall Furniture</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_44</key>
     <original>カーペットの種類</original>
+    <translation>Carpet Type</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_45</key>
     <original>家具</original>
+    <translation>Furniture</translation>
   </message>
   <message>
     <key>FURNITURE_LAYOUT_NAME_46</key>
     <original>呼び鈴</original>
+    <translation>Doorbell</translation>
```

Comment:
I think perhaps Summoning Bell is a better translation (I took the name from FF14, that is used to call your retainers)